docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -0,0 +1,65 @@
+description: Configuring NAT with Hyperledger Besu
+<!--- END of page meta data -->
+
+# Configuring NAT 
+
+Use the [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method) option to specify the 
+NAT method. Options are: `UPNP`, `MANUAL`, `DOCKER`, `AUTO`, and `NONE`. 
+
+The [enode](../../Concepts/Node-Keys.md#enode-url) advertised to other nodes during discovery is the 
+external IP address and port. The [`admin_nodeInfo`](../../Reference/API-Methods.md#admin_nodeinfo) 
+JSON-RPC API method returns the external address and port for the `enode` and `listenAddr` properties. 
+  
+While Hyperledger Besu is running, the following are not supported: 
+
+* IP address changes
+* Changing NAT methods. To change NAT method, restart the node with the [`--nat-method`](../../Reference/CLI/CLI-Syntax.md#nat-method)
+option set as required. 
+
+# UPnP
+
+Specify `UPNP` to quickly allow inbound peer connections without manual router configuration. Use UPnP 
+in home or small office environments where a wireless router or modem provides NAT isolation. 
+
+UPnP automatically detects that a node is running in a UPnP environment and provides port forwarding. 
+UPnP might introduce delays during node startup, especially on networks where no UPnP gateway device can be found.
+
+!!! tip 
+    UPnP support is often disabled by default in networking firmware. If disabled by default, explicitly
```

Comment:
Not sure what the convention is here for whether we have a blank line between !!! tip (or note, or example, etc.) and the text or not. I've seen it as both, and I've been including a blank line in my mklint and vale update PRs.

Reply:
It renders exactly the same way with or without the blank line so I think it's fine either way